packages/ui-billboard/src/Billboard/props.ts

```diff
@@ -113,19 +113,19 @@ const propTypes: PropValidators<PropKeys> = {
 }
 
 const allowedProps: AllowedPropKeys = [
+  'hero',
```

Comment:
I know you just reordered these, but was there a reason to it? Just curious, because when I started the allowedProps refact, in the beginning I started ordering them alphabetically, but gave up on it later... I hope I didn't break something with it.

Reply:
I just ordered them the same way Props are ordered, so its easy to check that they are the same